### DISCUSSION TITLE
Dealing with methods of same name but different parameters

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [vfsfitvnm/frida-il2cpp-bridge#462] [Q&A] [answered]
<pre>
System.Void InitiatePurchase(System.String productId); // 0x01ae5404
System.Void InitiatePurchase(UnityEngine.Purchasing.Product product, System.String developerPayload); // 0x01ae55e0
System.Void InitiatePurchase(System.String purchasableId, System.String developerPayload); // 0x01ae5468
</pre>

I want to implement anyone of them and then invoke them too. How can I do it?

**Comment by thinhbuzz** [comment, marked as answer]
Use overload, eg:
method('InitiatePurchase').overload(System.String')

**Reply by RevealedSoulEven** [reply]
Thanks a lot. What about multiple parametric functions?
